Player.py

```diff
@@ -101,3 +112,20 @@ def move(self):
             elif self.state >= 40:
                 self.state = 0
 
+        if keys[pygame.K_SPACE]:
+            if self.bombNumber > 0 and self.bombQueue == 1:
+                self.bombNumber -= 1
+                self.newBomb = Bomb(self.x, self.y, self.bombPower)
+                self.bombQueue = 0
+        else:
+            self.bombQueue = 1
+
+    def dictionaryToBomb(self, dictBomb):
+        x = dictBomb["x"]
+        y = dictBomb["y"]
+        power = dictBomb["power"]
+        self.bombs.append(Bomb(x, y, power))
+
+    def bombToDictionary(self, bomb):
+        return {"x": bomb.x, "y": bomb.y, "power": bomb.power}
```

Comment:
Shouldn't it be in `Bomb.py` class?

Reply:
These methods are mostly used in communication between server and player, so I left it in Player.py, but maybe it needs more inquiry